src/root.component.tsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import { useHistory } from 'react-router';
```

Comment:
Shift to the imports of `react-router-dom` (we exclusively use `react-router-dom` and not `react-router` - the latter is part of `react-router-dom`, but we do not share it explicitly)